Header header
octomap_msgs/Octomap[] octomaps
uint8 num_octomaps
